firebase-sessions/firebase-sessions.gradle.kts

```diff
@@ -42,12 +42,17 @@ android {
 }
 
 dependencies {
+  implementation(project(":encoders:firebase-encoders"))
+  implementation(project(":encoders:firebase-encoders-json"))
   implementation(project(":firebase-common"))
```

Comment:
Since we merged pinned deps on master, it'd be worthwhile to convert these to pinned deps.

Reply:
Done